math/benches/iai_field.rs

```diff
@@ -1,48 +1,94 @@
 use criterion::black_box;
-use util::rand_field_elements_pair;
+use lambdaworks_math::field::{
+    element::FieldElement, fields::fft_friendly::stark_252_prime_field::Stark252PrimeField,
+};
 
 mod util;
 
 #[inline(never)]
 fn fp_add_benchmarks() {
-    let (x, y) = rand_field_elements_pair();
+    let x = FieldElement::<Stark252PrimeField>::from_hex(
+        "0x03d937c035c878245caf64531a5756109c53068da139362728feb561405371cb",
+    )
+    .unwrap();
+    let y = FieldElement::<Stark252PrimeField>::from_hex(
+        "0x0208a0a10250e382e1e4bbe2880906c2791bf6275695e02fbbc6aeff9cd8b31a",
+    )
+    .unwrap();
```

Comment:
Extract this somewhere. This is pure noise in the measurement, the original function was filtered out by callgrind.